core/src/main/scala/akka/persistence/cassandra/ConfigSessionProvider.scala

```diff
@@ -109,7 +109,7 @@ class ConfigSessionProvider(system: ActorSystem, config: Config) extends Session
   def clusterBuilder(clusterId: String)(implicit ec: ExecutionContext): Future[Cluster.Builder] = {
```

Comment:
We need the id-atomics because this `clusterId` will be the same for multiple instances?

Reply:
yeah we create one for snapshot, read and writes. Ideally we'd just create one but it would be a large rework